lib/clear.js

```diff
@@ -0,0 +1,169 @@
+import {
+    client,
+    categoriesService,
+    ordersService,
+    stateService,
+    customersService,
+    shippingMethodService,
+    zonesService,
+    productsService
+} from './services'
+
+const deleteCategories = () => {
+    client.execute({
```

Comment:
client.process(...)  is your friend for iterating through everything  https://commercetools.github.io/nodejs/sdk/api/sdkClient.html#processrequest-processfn-options 

The default is 20 items returned, that does not even suffice here. 

An alternative when deleting is to use GraphQL to fetch _only_ the ids and versions to be able to brute-force delete. You don't really need anything else from the API payload and that way you can safely fetch 500 or 1000 in one call.